packages/consent-manager/index.test.js

```diff
@@ -49,42 +49,19 @@ const defaultProps = {
     },
   ],
   container: '#consent-manager',
-  forceShow: true,
```

Comment:
`forceShow` is a "dev only" prop, so having it be in default props here didn't make sense to me. It's manually set on almost every test anyways.